docs/installing.rst

```diff
@@ -247,5 +265,26 @@ There are two ways to download the ADB.
     # Add the image to vagrant
     $ vagrant box add adb <local path to the downloded image>
 
+--------------------------------
+Setting up ADB behind http proxy
+--------------------------------
+
+ADB can be setup behind a proxy server. You need to export the proxy server
+information in to the environment and then run ``vagrant up``.
+
+**Note:** Currently, only HTTP and HTTPS proxy servers are supported.
+
+For GNU/Linux, OS X and Cygwin shell::
```

Comment:
I think we can just say Linux and not GNU/Linux.  Also, I believe we should clarify it is Windows using Cygwin.